imports/api/crud-controller/module/methods.js

```diff
@@ -75,6 +101,12 @@ export const getOneModuleInSemester = function getOneModuleInSemester(semesterIn
   if (!retrievedSemesters) {
     return {};
   }
+
+  // checks semester index is not larger than the size of semester
```

Comment:
What does this check for again? Or perhaps this is defensive programming in case the index of a sem is larger than its actual size?